src/main/java/net/minecraftforge/event/item/SweepHitBoxEvent.java

```diff
@@ -0,0 +1,59 @@
+package net.minecraftforge.event.item;
+
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.phys.AABB;
+
+/**
+ * Event that modifies the Sweeping Edge Hit Box of items
+ */
+public class SweepHitBoxEvent extends PlayerItemStackEvent {
+
+	private final Entity target;
+	private final AABB originalHitBox;
+	private AABB resultHitBox;
+
+	public SweepHitBoxEvent(Player player, ItemStack stack, Entity target, AABB hitBox) {
+		super(player, stack);
+		this.target = target;
+		this.originalHitBox = hitBox;
+		this.resultHitBox = hitBox;
+	}
+
+	/**
+	 * get the attacked entity
+	 */
+	public Entity getTarget() {
+		return target;
+	}
+
+	/**
+	 * get the original hit box provided by the Item
```

Comment:
Use `@return` instead.
```suggestion
	 * @return the original hitbox as given by the item
```